test/pdfinfo.test.js

```diff
@@ -52,13 +52,22 @@ describe('pdfinfo', function(){
   });
 
 
-  describe('error', function(done){
```

Comment:
I'm not even sure if this even worked before since I thought `done`s has to be stuck onto `it` blocks only.

Reply:
@et Thanks for catching it :+1: